### PR TITLE
Testing/app

### PR DESCRIPTION
**Pull request by laurenlucero**
## What does this PR do?
Tests for SignIn, AreaCard, AreasContainer and Favorites complete, pending review
PropTypes for SignIn, AreaCard, AreasContainer and Favorites started, need review
App tests started

## Where should the reviewer start? (include files modified)
App.test.js
SignIn.js / SignIn.test.js
AreaCard.js / AreaCard.test.js
AreasContainer.js / AreasContainer.test.js
Favorites.js / Favorites.test.js

## How should this be manually tested?
Review PropTypes in component files for accuracy and tests in test files
Run npm test (all passing at time of PR)

## Any background context you want to provide?
n/a

## What are the relevant tickets?
n/a

## Screenshots (if appropriate)
n/a

## Questions?
What do you think of PropTypes and testing thoroughness?

### CODE REVIEW

File: src/components/AreaCard/AreaCard.js [hunk 1]
```diff
@@ -1,22 +1,24 @@
...
       </Link>
     </section>
   );
 };
 
 export default AreaCard;
 
-AreaCard.propTypes = {};
+AreaCard.propTypes = {
+  setCurrentArea: PropTypes.func.isRequired,
```
Comment (pernstrong) [inline src/components/AreaCard/AreaCard.js:23]: Maybe test for the area object as well?

File: src/components/AreasContainer/AreasContainer.js [hunk 2]
```diff
@@ -1,19 +1,27 @@
...
+    );
+  });
+  return <section className="areas-container">{areas}</section>;
 };
 
 export default AreasContainer;
 
-AreasContainer.propTypse = {};
+AreasContainer.propTypse = {
+  areas: PropTypes.array.isRequired,
```
Comment (pernstrong) [inline src/components/AreasContainer/AreasContainer.js:25]: This looks great!

File: src/components/Favorites/Favorites.test.js [hunk 3]
```diff
@@ -0,0 +1,57 @@
...
+      <Router>
+        <Favorites favorites={mockFavorites} />
+      </Router>
+    );
+    expect(getByText("My Favorites")).toBeInTheDocument();
+    expect(
+      getByText("You have no favorites yet. Favorite a listing!")
+    ).toBeInTheDocument();
+  });
+
```
Comment (pernstrong) [inline src/components/Favorites/Favorites.test.js:22]: Maybe test the button functionality for Return to Listings as well?

Comment (laurenlucero) [reply src/components/Favorites/Favorites.test.js:22]: Would this be an integration test in App because we are going back to another component?

**Pull request merged**